### ISSUE TITLE
Suspicious code

### ISSUE THREAD

**Issue by AnyhowStep** [kripken/sql.js#294]
https://github.com/kripken/sql.js/blob/4e91be7da52cdcf26fabb10fcc6f922d3a823cb2/dist/worker.sql-wasm.js#L272

This `err` is assigned to but never read.

https://github.com/kripken/sql.js/blob/4e91be7da52cdcf26fabb10fcc6f922d3a823cb2/dist/worker.sql-wasm.js#L279

This is essentially,
```js
throw new "hello" + "world";
```

Which gives,
```
Uncaught TypeError: "hello" is not a constructor
```

**Comment by lovasoa** [comment]
Good catch, thank you. The error comes from here : https://github.com/kripken/sql.js/blob/4e91be7da52cdcf26fabb10fcc6f922d3a823cb2/src/worker.coffee#L51

You can open a PR !

**Comment by bryangingechen** [comment]
This is addressed by #288.
